src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/Schemas/JSON/template.json

```diff
@@ -481,11 +459,7 @@
                 "type": "object"
               },
               "condition": {
-                "description": "A C++ style expression that, if it evaluates to 'false' causes the post-action to be skipped. This expression may refer to any symbols that have been defined",
-                "type": "string"
-              },
-              "configFile": {
-                "description": "Additional configuration for the associated post action. The structure & content will vary based on the post action.",
+                "description": "A C++ style expression that, if it evaluates to 'false' causes the post-action to be skipped. This expression may refer to any symbols that have been defined.",
                 "type": "string"
               },
               "continueOnError": {
```

Comment:
We allow only bool in json, but accept a string in code as well.

Thinking if we should make the code more restrictive - as it would start failing anyways if we move to anotation based deserialization or strict schema enforcement.

Another option is do declare support for `"true"`, `"false"` strings in json.

Or we can just keep as is - and then in future have a 'lenient' and 'strict' modes and fail only in later one